### PR TITLE
18161 Updated in response to UXA notes

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:* /bcgov/entity#18161

*Description of changes:*
- Fixed error validations triggering although text-fields are valid.
Yui's comment: https://github.com/bcgov/entity/issues/18161#issuecomment-1792806349

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/AgmLocationChange/AgmLocation.vue [hunk 1]
```diff
@@ -52,6 +52,7 @@ export default class AgmLocation extends Vue {
   /** Emits an event with the changed input (ie, updated v-model). */
   @Emit('update:agmLocation')
   emitInput (val: string): void {
+    this.agmLocation = val
```
Comment (JazzarKarim) [inline src/components/AgmLocationChange/AgmLocation.vue:55]: I noticed that the agmLocation variable here wasn't being changed. This was causing the function on line 45 to ALWAYS trigger. Hence, the fields were always showing the they're not valid although they are just as Yui mentioned:

yui comment agm location

After my update:
after my update yui

It's the exact same thing for the other components.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-567-7taa5hlg.web.app

**Review by severinbeauvais — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-567-7taa5hlg.web.app

**Comment by severinbeauvais** [comment]
Yui moved the ticket to RFQA so we'd better merge this :)

**Review by ketaki-deodhar — APPROVED** [review]

**Pull request merged**